strapi/strapi
# chore(content-releases): check license when registering plugin

### What does it do?

Check the project license when registering the content releases plugin because this is a EE feature.

### How to test it?

You need to have two different licenses, one with releases enabled and other with no releases enabled.

In case you have releases enabled you should see the message on the console when running the server. Otherwise, you should not see any message related to content releases.
